### ISSUE TITLE
About CTC

### ISSUE THREAD

**Issue by thunder123321** [FLC777/GLAT#19]
I'd like to ask you something. Can I calculate loss with cross entropy loss on the generated ctc_align sequence instead of torch.nn.CTCloss() in the code？
